test/groovy/ToolValidateTest.groovy

```diff
@@ -47,43 +57,7 @@ class ToolValidateTest extends BasePipelineTest {
         binding.setVariable('JAVA_HOME', home)
 
         toolValidateScript =  loadScript('toolValidate.groovy').toolValidate
-    }
-
-
-    @Test
-    void nullHomeTest() {
-
-        thrown.expect(IllegalArgumentException)
-        thrown.expectMessage("The parameter 'home' can not be null or empty.")
-
-        toolValidateScript.call(tool: 'java', home: null)
-    }
-
-    @Test
-    void emptyHomeTest() {
-
-        thrown.expect(IllegalArgumentException)
-        thrown.expectMessage("The parameter 'home' can not be null or empty.")
-
-        toolValidateScript.call(tool: 'java', home: '')
-    }
-
-    @Test
-    void nullToolTest() {
```

Comment:
just as remark: nullToolTest here is replaced by toolNotSetTest below, which is quite similar.